cts/lab/CTStests.py

```diff
@@ -1094,20 +1095,6 @@ def __call__(self, node):
 
         return self.success()
 
-    def NextIP(self):
-        ip = self._env["IPBase"]
-        if ":" in ip:
-            fields = ip.rpartition(":")
-            fields[2] = str(hex(int(fields[2], 16)+1))
-            print(str(hex(int(f[2], 16)+1)))
-        else:
-            fields = ip.rpartition('.')
-            fields[2] = str(int(fields[2])+1)
-
-        ip = fields[0] + fields[1] + fields[3];
-        self._env["IPBase"] = ip
-        return ip.strip()
-
```

Comment:
I've never used BSC / BasicSanity. Compared to the default scenario, it looks like it just adds a couple resources (less intrusive than --clobber-cib). I don't see much value in that over simply using the existing configuration, though. I'd be fine with dropping it all.

Reply:
Are you thinking the BSC scenario, the BSC_AddResource test, the DoBSC environment setting, all of it?

Reply:
All of it. I haven't really taken advantage of the Scenario feature, other than random tests and specifically named tests, so it might be more useful than it appears, but I don't think the lab gets much use by anyone but us.

Reply:
I've added a patch to remove the BSC_AddResource test.  I'll take care of the rest when I look at scenarios, which I think is the next piece to work on.  I was going to dive into them more and see how well they work.

Reply:
Looking at the scenarios more closely... they are not really what I thought they would be.  In addition to the random and specifically named tests, I've been using AllOnce a lot in testing these cts changes.  Beyond that, I don't think any are actually used - LeaveBooted requires the --boot command line option.  PingFest, Benchmark, and RollingUpgrade aren't hooked up anywhere.

Reply:
Let's drop BasicSanity, PingFest, Benchmark, and RollingUpgrade

Reply:
Sure, I'll drop those when I'm working on scenarios.